client/layout/index.jsx

```diff
@@ -250,7 +249,7 @@ class Layout extends Component {
 
 		return (
 			<div className={ sectionClass }>
-				<SidebarScrollSynchronizer enabled={ this.props.isNavUnificationEnabled } />
+				<SidebarScrollSynchronizer />
```

Comment:
```suggestion
				<SidebarScrollSynchronizer enabled={ ! config.isEnabled( 'jetpack-cloud' ) } />
```

How would you feel add the `Jetpack Cloud` check here since it uses the old sidebar which doesn't scroll together with the content?

Reply:
I don't think it's needed, since it's already handled by the `SidebarScrollSynchronizer` component:

https://github.com/Automattic/wp-calypso/blob/acacd8f21ee4b0e1f1e06c391d0db054c22ef428/client/layout/index.jsx#L51